package.json

```diff
@@ -22,9 +22,9 @@
     },
     "homepage": "https://github.com/andela/pippin-ah-frontend#readme",
     "dependencies": {
+        "express": "^4.16.4",
```

Comment:
why did you move this up?

Reply:
No concrete reason, just that it occured twice so I deleted one and moved the other